fe/fe-core/src/main/java/com/starrocks/transaction/TransactionState.java

```diff
@@ -343,6 +344,20 @@ public String toString() {
     private ConcurrentMap<String, TOlapTablePartition> partitionNameToTPartition = Maps.newConcurrentMap();
     private ConcurrentMap<Long, TTabletLocation> tabletIdToTTabletLocation = Maps.newConcurrentMap();
 
+    private final ReentrantReadWriteLock txnLock = new ReentrantReadWriteLock(true);
```

Comment:
why don't use lock manager to manage txn state lock?

Reply:
Here I have not considered whether all locks are managed by LockManager, so currently only the metadata at the db-table level is managed by LockManager. It can be easily modified later if necessary.